src/Cpp/1-getting-started/2-HelloFroge/Main.cpp

```diff
@@ -24,11 +24,7 @@ using Microsoft::WRL::ComPtr;
 #include <vector>
 #include <unordered_map>
 
-#pragma comment(lib, "d3d11.lib")
-#pragma comment(lib, "dxgi.lib")
-#pragma comment(lib, "d3dcompiler.lib")
-#pragma comment(lib, "winmm.lib")
-#pragma comment(lib, "dxguid.lib")
+#include "../Shared/CommonLibraries.h"
```

Comment:
Hmm I am not sure if I like that.

HelloFroge is going to be removed anyway :)